playbooks/packstack.yaml

```diff
@@ -1,13 +1,16 @@
 ---
 - hosts: packstack
+  gather_facts: yes
```

Comment:
gather_facts is defaulted to yes and updated on every play - is there a reason why there is a gather_facts here ?